pkg/pool-gyro/test/foundry/E2eSwap2CLP.t.sol

```diff
@@ -39,13 +39,15 @@ contract E2eSwapGyro2CLPTest is E2eSwapTest, Gyro2ClpPoolDeployer {
         sender = lp;
         poolCreator = lp;
 
+        E2eTestState memory state = _getTestState();
         // 0.0001% min swap fee.
-        minPoolSwapFeePercentage = 1e12;
+        state.minPoolSwapFeePercentage = 1e12;
         // 10% max swap fee.
-        maxPoolSwapFeePercentage = 10e16;
+        state.maxPoolSwapFeePercentage = 10e16;
+        _setTestState(state);
     }
 
-    function calculateMinAndMaxSwapAmounts() internal virtual override {
+    function calculateMinAndMaxSwapAmounts() internal view virtual override returns (SwapAmounts memory swapAmounts) {
```

Comment:
```suggestion
    function computeSwapLimits() internal view virtual override returns (SwapLimits memory swapLimits) {
```

Anticipating Jeff's comment :p . We're using compute, and maybe swap limits makes more sense.